ex_webrtc/integration_test/test_videoroom/lib/test_videoroom/room.ex

```diff
@@ -9,37 +9,34 @@ defmodule TestVideoroom.Room do
   require Logger
 
   def start(opts) do
-    GenServer.start(__MODULE__, [], opts)
+    GenServer.start(__MODULE__, nil, opts)
   end
 
   def start_link(opts) do
-    GenServer.start_link(__MODULE__, [], opts)
+    GenServer.start_link(__MODULE__, nil, opts)
   end
 
-  def add_peer_channel(room, peer_channel_pid, peer_id) do
-    GenServer.call(room, {:add_peer_channel, peer_channel_pid, peer_id})
+  def add_peer_channel(room, peer_channel_pid, peer_id, video_codec) do
+    GenServer.call(room, {:add_peer_channel, peer_channel_pid, peer_id, video_codec})
   end
 
   def register_new_peer_listener(room, listener) do
     GenServer.call(room, {:register_new_peer_listener, listener})
   end
 
   @impl true
-  def init(room_id) do
+  def init(nil) do
     Logger.info("Spawning room process: #{inspect(self())}")
-
-    rtc_engine_options = [id: room_id]
-
-    {:ok, pid} = Membrane.RTC.Engine.start(rtc_engine_options, [])
+    
+    {:ok, pid} = Membrane.RTC.Engine.start([id: ""], [])
```

Comment:
```suggestion
    {:ok, pid} = Membrane.RTC.Engine.start([], [])
```
or nil instead of list